### DISCUSSION TITLE
fail with code 404 when request to token uri/token_instances

### DISCUSSION THREAD

**Discussion by BlockgenStudio** [blockscout/blockscout#9602] [Q&A] [answered]
I am trying to deploy block scout for a polygon edge private appchian
```
Checkout git@github.com:blockscout/blockscout.git
cd docker-compose
edit envs/common-blockscout.env:
ETHEREUM_JSONRPC_VARIANT=geth
ETHEREUM_JSONRPC_HTTP_URL=our polygon-edge json-rpc endpoint
ETHEREUM_JSONRPC_TRACE_URL=our polygon-edge json-rpc endpoint
JSON_RPC=our polygon-edge json-rpc endpoint
CHAIN_ID=our chain id
DATABASE_URL=postgresql://blockscout:ceWb1MeLBEeOIfk65gU8EjF8@db:5432/blockscout
INDEXER_INTERNAL_TRANSACTIONS_TRACER_TYPE=polygon_edge
4. run docker-compose -f ./geth.yml up
```

we get the following error: 
`backend          | {"time":"2024-03-11T14:58:07.453Z","severity":"warn","message":"Request to token uri: \"https://cert.client.org/xxxxxxxxxxxx\" failed with code 404. Body:\"\"","metadata":{"fetcher":"token_instances"}}`

any idea what went wrong and how to fix this?

**Comment by sl1depengwyn** [comment, marked as answer]
`https://cert.client.org` responses 404 when Blockscout tries to request NFT metadata, this is actually not an error, but warning as it happens quite often

You can ignore it, the NFT image won't appear in Blockscout till contract developer fix the issue
